tracing-subscriber/src/registry/mod.rs

```diff
@@ -5,7 +5,7 @@
 //! This module provides the [`Registry`] type, a [`Collect`] implementation
 //! which tracks per-span data and exposes it to subscribers. When a `Registry`
 //! is used as the base `Collect` of a `Subscribe` stack, the
-//! [`layer::Context`][ctx] type will provide methods allowing subscribers to
+//! [`subscriber::Context`][ctx] type will provide methods allowing subscribers to
```

Comment:
wrong module name here, it's now `subscribe`, rather than `subscriber`
```suggestion
//! [`subscribe::Context`][ctx] type will provide methods allowing subscribers to
```